TrafficCapture/testingEnvironment/build.gradle

```diff
@@ -0,0 +1,125 @@
+plugins {
+    id 'org.opensearch.migrations.java-library-conventions'
+    id "com.avast.gradle.docker-compose" version "0.16.12"
+    id "com.bmuschko.docker-java-application" version "9.3.1"
+}
+
+import java.security.MessageDigest
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import org.apache.tools.ant.taskdefs.condition.Os
+
+def calculateDockerHash(String projectName) {
```

Comment:
This should be refactored into a common shared class and imported by both projects